.github/workflows/build.yml

```diff
@@ -416,167 +56,34 @@ jobs:
             dvc/pyproject.toml
             dvc-gs/pyproject.toml
       - name: install reqs
-        run: uv pip install "./dvc[tests]" "./dvc-gs[tests]"
+        run: pip install "./dvc[tests]" "./dvc-gs[tests]"
       - uses: actions/cache/restore@v4
         with:
           path: dvc-bench/.dvc/cache
           key: ${{ env.DATASET }}
       - id: 'auth'
-        if: ${{ github.event_name == 'schedule' }}
         name: 'Authenticate to GCP'
         uses: 'google-github-actions/auth@v2.1.5'
         with:
-          create_credentials_file: true
```

Comment:
@0x2b3bfa0, dvc needs a credentials file (and `GOOGLE_APPLICATION_CREDENTIALS` to be set).

Reply:
I deleted that option because it defaults to `true` 🤪

Reply:
I am not sure how the CI is working though.

Fixtures do require it. https://github.com/iterative/dvc-gs/blob/b0179fb272b03b8f4f6b5f83c5640b6e083cf779/dvc_gs/tests/fixtures.py#L13

Reply:
Ahh, okay. I did not know that.